pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -332,9 +350,14 @@ func (c *Cluster) makeDeployment(osdProps osdProperties, osd OSDInfo, provisionC
 
 	if osdProps.onPVC() && osd.CVMode == "raw" {
 		if osdProps.encrypted {
-			// Add a new init container to open the encrypted disk!
+			// Open the encrypted disk
 			initContainers = append(initContainers, c.getPVCEncryptionOpenInitContainerActivate(osdProps)...)
+			// Copy the encrypted block to the osd data location, e,g: /var/lib/ceph/osd/ceph-0/block
 			initContainers = append(initContainers, c.getPVCEncryptionInitContainerActivate(osdDataDirPath, osdProps)...)
+			// Print the encrypted block status
+			initContainers = append(initContainers, c.getEncryptedStatusPVCInitContainer(osdDataDirPath, osdProps))
```

Comment:
Is there a unit test that covers creating the encrypted pod spec?

Reply:
Not really. I can take care of that.

Reply:
Actually, I've added a few during the initial implementation, I can try to add more.

Reply:
Added!